src/Symfony/Contracts/HttpClient/Exception/ClientExceptionInterface.php

```diff
@@ -18,6 +18,6 @@
  *
  * @experimental in 1.1
  */
-interface ClientExceptionInterface extends ExceptionInterface
+interface ClientExceptionInterface extends ExceptionInterface, HttpExceptionInterface
```

Comment:
`ExceptionInterface` should be removed as it's now embeded in HttpExceptionInterface
same in the 2 other exception interfaces